File: src/main/java/com/fwdekker/randomness/uuid/UuidSettingsAction.java
package com.fwdekker.randomness.uuid;

import com.fwdekker.randomness.SettingsAction;
import com.fwdekker.randomness.SettingsDialog;


/**
 * Controller for random UUID generation settings.
 */
public final class UuidSettingsAction extends SettingsAction {
    @Override
    public SettingsDialog createDialog() {
        return new UuidSettingsDialog();
    }

    @Override
    public String getTitle() {
        return "Integer Settings";
    }
}

File: src/main/kotlin/com/fwdekker/randomness/uuid/UuidSettingsAction.kt
package com.fwdekker.randomness.uuid

import com.fwdekker.randomness.SettingsAction


/**
 * Controller for random UUID generation settings.
 */
class UuidSettingsAction : SettingsAction() {
    override val title = "UUID Settings"


    public override fun createDialog() = UuidSettingsDialog()
}
